app/main.py

```diff
@@ -1,8 +1,51 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class if 1 <= comfort_class <= 7 else 1
```

Comment:
Good job on adding data validation. Later in the course we will learn how to make it even better :+1: